src/aluno/PerfilPage.tsx

```
@@ -0,0 +1,213 @@
+            //const token = localStorage.getItem('token')
+
+            setUsuario((prev) => prev && { ...prev, ...data })
+            toast.success('Dados atualizados com sucesso!')
+            setEditDialogOpen(false)
+        } catch (error) {
+            toast.error('Erro ao atualizar perfil.')
+            console.error(error)
+        }
+    }
+
+    useEffect(() => {                                                //Deve ser revisado após a implementação do serviço de usuário
```

ESLint

[error] Replace `··useEffect(()·=>·{············································` with `useEffect(()·=>·{⏎`

(prettier/prettier)